dashboard/src/presenters/toWorldMap.ts

```diff
@@ -42,8 +47,19 @@ export function dataToWorldMap(data: GeoVisitor[], format: CountryCodeFormat): W
     return updatedCountryData;
   });
 
+  const processedCompareData = compareData.map((visitor) => {
+    const transformedData = transformerFunction(visitor.country_code);
+    return transformedData
+      ? {
+          ...visitor,
+          country_code: transformedData,
+        }
+      : visitor;
+  });
```

Comment:
I'm a little confused by this 😄 Why can the `transformerFunction` return something nullish? And doesn't this result in an array of data with formats of country codes?

Reply:
Perfect, then it's all good 😎👌